spec/tic_tac_toe_spec.rb

```diff
@@ -1,15 +1,38 @@
-# require "rspec/autorun"
-require_relative "../lib/tic_tac_toe.rb"
+require_relative '../bin/main.rb'
+require_relative '../lib/tic_tac_toe.rb'
+require_relative '../lib/player.rb'
```

Comment:
You don't have any tests for player class. It's a small class but you can test `initialize` method.

Reply:
Thanks for your comments! We tested the initialize.